modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,57 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Optional: Enabling monitoring of event listeners for Triggers for user-defined projects
+
+On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`. Cluster administrators can configure the monitoring of event listener metrics for Triggers in a user-defined project. When configured, users and administrators can view the metrics in the OpenShift web console.
```

Comment:
From the checklist: "Check for places where `OpenShift` should be `{product-title}`"
How about:
```suggestion
On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`. Cluster administrators can configure the monitoring of event listener metrics for Triggers in a user-defined project. When configured, users and administrators can view the metrics in the {product-title} web console.
```
or
```suggestion
On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`. Cluster administrators can configure the monitoring of event listener metrics for Triggers in a user-defined project. When configured, users and administrators can view the metrics in the web console.
```

Reply:
```suggestion
As a cluster administrator, to gather event listener metrics for Triggers in a user-defined project and display them in the web console, you can create a service monitor for each event listener. On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`.
```

Reply:
In response to removing "Optional:" from the title, please consider this ^^ alteration of the abstract, which puts the user story and condition first.